core/storage/collection/gae_models.py

```diff
@@ -493,11 +472,10 @@ def get_deletion_policy():
 
     @staticmethod
     def get_export_policy():
-        """This model's export_data function implementation is still pending.
-
-        TODO(#8523): Implement this function.
+        """The history of commits is not relevant to a user for the purposes of
+        Takeout.
         """
-        return base_models.EXPORT_POLICY.TO_BE_IMPLEMENTED
+        return base_models.EXPORT_POLICY.NOT_APPLICABLE
```

Comment:
Why we don't return the content of the commit when the user is author of the commit?

Reply:
I spoke with Sean about this and we concluded that it's not relevant for the purposes of Takeout.